lib/game.rb

```diff
@@ -8,6 +8,10 @@ def initialize(board, checker, player1, player2)
   end
 
   def play
+    trap('INT') do
```

Comment:
Should the `Game` know about trapping signals?

Reply:
Moved these for the app to handle.